src/views/Help/index.js

```diff
@@ -2,256 +2,263 @@ import React, { Component } from 'react';
-            <br />
-            <Grid item>
-              <Card>
-                <div style={headerStyle}>
-                  <Typography variant="body2" style={headerStyle}>
-                    FAQ
-                  </Typography>
-                </div>
-              </Card>
-            </Grid>
-            <Typography variant="body2" gutterBottom component="ul">
-              <br />
-              <li>
-                <strong>Do I have to rejoin an Involvement each semester?</strong> Yes. This allows
-                each student to control their inclusion in Involvements for their Co-Curricular
-                Transcript. Student Leaders and Advisors can ADD members and guests, too, and some
-                may do this to help maintain their team roster.
-              </li>
-              <li>
-                <strong>
-                  What&apos;s the difference between subscribing to, and joining an Involvement?{' '}
-                </strong>
-                Subscribing to an Involvement allows you to receive emails from the Involvement, but
-                not to view other members of the Involvement. Joining a Involvement means you're a
-                member of the Involvement, and can view other members in the Involvement, in
-                addition to getting emails.
-              </li>
-              <li>
-                <strong>What if I leave an Involvement?</strong> Subscribership and Membership are
-                fluid during an academic session, and are confirmed at the conclusion of a session.
-                You can come and go over the semester, and your Membership will move to your
-                official Co-Curricular Transcript only at the end of the semester.
-              </li>
-              <li>
-                <strong>What if I am in an Involvement I don&apos;t want to be part of? </strong>
-                No problem. Contact the Involvement leaders or advisor.
-              </li>
-              <li>
-                <strong>
-                  I&apos;m an alumna or alumnus. Why do I not see the join button anymore?{' '}
-                </strong>
-                Sorry, but alumni cannot join new Involvements. You could still subscribe to
-                Involvements to receive emails!
-              </li>
-            </Typography>
+          <Grid item xs={12} lg={8}>
+            <Card className="help">
+              <CardHeader
+                className="help-title"
+                title="Gordon 360 Help"
+                titleTypographyProps={{ variant: 'h4' }}
+              />
+              <CardContent>
+                <Card>
+                  <CardHeader
+                    className="help-header"
+                    title={`LOGIN INSTRUCTIONS`}
+                    titleTypographyProps={{ variant: 'body1' }}
+                  />
+                  <CardContent>
+                    <Typography variant="body1" component="ul" style={{ textAlign: 'start' }}>
```

Comment:
Is there a reason you decided to use an in-line style for all of these instances? Since this is related to the timesheets text alignment bug, these should be obsolete eventually anyways.